PROJECT_IDEA_TEMPLATE.md

```diff
@@ -18,5 +18,5 @@
 
 - Description: Currently the HTTP API is not very well organized and needs some tidying up. The actual course of action is not decided yet, but [go-kit](https://github.com/go-kit/kit) looks like a good fit.
 - Recommended Skills: golang
-- Mentor(s): Krasi Georgiev (@krasi-georgiev)
+- Mentor(s): Krasi Georgiev (@krasi-georgiev) - primary, John Doe(@johndoe)
```

Comment:
I'm a bit confused by this format.  Let's update both the template and the sample with what we want it to look like:

```suggestion
- Mentor(s):
  - Jane Doe (@jane-github) - primary
  - John Doe (@john-github)
```

(we should find some better names...)